tests/e2e/e2e_test.go

```diff
@@ -161,10 +161,9 @@ func (s *IntegrationTestSuite) TestIBCTokenTransferRateLimiting() {
 	fmt.Println(wd, projectDir)
 	err = copyFile(projectDir+"/x/ibc-rate-limit/bytecode/rate_limiter.wasm", wd+"/scripts/rate_limiter.wasm")
 	s.NoError(err)
-	// set LatestCodeId to 1 since we upload a contract in the upgrade handler for v13
-	chainA.LatestCodeId = 1
+
 	node.StoreWasmCode("rate_limiter.wasm", initialization.ValidatorWalletName)
-	chainA.LatestCodeId += 1
+	chainA.LatestCodeId = 1
```

Comment:
@czarcas7ic  This looks good, but similar question. Why not cherry pick https://github.com/osmosis-labs/osmosis/pull/3635/?

Reply:
I think my main concern was I didn't want to cherry-pick a PR and not realize it had some drive by that it shouldn't have